Tests/PersonalityInsightsV3Tests/PersonalityInsightsTests.swift

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import XCTest
+import PersonalityInsightsV3
+
+class PersonalityInsightsTests: XCTestCase {
+
+    private var personalityInsightsV3: PersonalityInsights!
+    private var mobyDickIntro: String!
+    private var kennedySpeech: String!
+    private let timeout: TimeInterval = 5.0
+    private var version: String = "2016-10-20"
+    
+    static var allTests : [(String, (PersonalityInsightsTests) -> () throws -> Void)] {
+        return [
+            ("testProfile", testProfile),
+            ("testContentItem", testContentItem),
+            ("testProfileWithShortText", testProfileWithShortText)
+        ]
+    }
+
+    // MARK: - Test Configuration
+
+    /** Set up for each test by loading text files and instantiating the service. */
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+        instantiatePersonalityInsights()
+        loadMobyDickIntro()
+        loadKennedySpeech()
+    }
+
+    /** Instantiate Personality Insights. */
+    func instantiatePersonalityInsights() {
+        let username = Credentials.PersonalityInsightsV3Username
+        let password = Credentials.PersonalityInsightsV3Password
+        personalityInsightsV3 = PersonalityInsights(username: username, password: password)
```

Comment:
If you update the `version` to be an argument of `PersonalityInsights.init(...)` then you can just specify it here instead of making it a class property. Not particularly necessary, but I think that's consistent with the tests for other services.